server/views/pages/bookedtoday/summaryWithRecord.njk

```diff
@@ -24,6 +24,14 @@
                     <h1 class="govuk-heading-l">{{ arrival.lastName }}, {{ arrival.firstName }}</h1>
                 </div>
                 <div class="govuk-grid-column-one-half">
+                    {% if(enableDpsLink) %}
+                        {{ govukButton({
+                            text: "View full prisoner profile",
+                            href: dpsUrl + "/prisoner/" + arrival.prisonNumber,
```

Comment:
as mentioned yesterday, you need to go via the special endpoint to enable the link back to welcome people to prison

Reply:
the add case note link is a good example

Reply:
thank you. I had misunderstood not realising Connor had implemented code to remove the need to directly add a back link in DPS. 
Anyway, change above applied.